app/components/PublishingInfo.js

```diff
@@ -60,7 +65,13 @@ function PublishingInfo({
         archived <Time dateTime={archivedAt} /> ago
       </span>
     );
-  } else if (publishedAt && (neverUpdated || showPublished)) {
+  } else if (createdAt === updatedAt) {
+    content = (
+      <span>
+        created <Time dateTime={updatedAt} /> ago
+      </span>
+    );
+  } else if (publishedAt && (publishedAt === updatedAt || showPublished)) {
```

Comment:
in this else if clause, `publishedAt === updatedAt` can't possibly be true right?

Reply:
It's true in the instance that the last action taken was to publish the document

Reply:
oh I read this wrong, I thought it said `createdAt === updatedAt`